src/Core/src/Platform/IMauiContext.cs

```diff
@@ -11,7 +11,21 @@ public interface IMauiContext
 
 #if __ANDROID__
 		global::Android.Content.Context? Context { get; }
-#elif __IOS__
+#endif
+	}
+
+	public interface IMauiApplicationContext : IMauiContext
```

Comment:
A new `IMauiApplicationContext` for an application. This does not have UI features.